main.tex

```diff
@@ -998,31 +1122,88 @@ \section{Technician Consumables}
         \vspace{0.5\unitsize}}
 \end{tcbraster}
 % END Technician Consumables
-
 \tcbline
 
-\section{Electrification \& Symbols}
+\include{Tourist.tex}
```

Comment:
I tried to render this with `pdflatex`, the new pages appear blank. In the log, I saw a few import errors -

```
No file Tourist.tex.tex
...
No file enbesa.tex.tex.
No file shepards.tex.tex.
No file elders.tex.tex.
[5]
No file SkyGeneral.tex.tex.
No file SkyEngineers.tex.tex.
[6]
No file skyinvestors.tex.tex.
No file scholars.tex.tex.
[7]
No file recipes.tex.tex.
[8]
No file patents.tex.tex.
```

I was able to fix this by removing `.tex` from the new `\include{}` lines. For exmaple

`\include{Tourist.tex}` -> `\include{Tourist}`

After removing `.tex`, the new pages are populated with graphics :)

![image](https://user-images.githubusercontent.com/84053270/226142474-767333a9-79d5-47aa-bd20-0f329c09be97.png)